bluesky/callbacks/olog.py

```diff
@@ -141,7 +143,7 @@ def lbcb(name, doc):
 def call_str(start, call_template=None):
     """Given a start document generate an evalable call scring
 
-    The default template assumes that `plan_args` and `plan_type`
+    The default template assumes that `plan_args` and `plan_name`
```

Comment:
Hmm. How can we write a call string without args? This template is not used by default as far as I can tell, so a hard assumption here does not break the default logging config.

Get Outlook for iOS<https://aka.ms/o0ukef>




On Mon, Aug 1, 2016 at 5:49 PM -0400, "Thomas A Caswell" <notifications@github.com<mailto:notifications@github.com>> wrote:


In bluesky/callbacks/olog.py<https://github.com/NSLS-II/bluesky/pull/531#discussion_r73059470>:

> @@ -141,7 +143,7 @@ def lbcb(name, doc):
>  def call_str(start, call_template=None):
>      """Given a start document generate an evalable call scring
>
> -    The default template assumes that `plan_args` and `plan_type`
> +    The default template assumes that `plan_args` and `plan_name`


No longer make strict assumption about 'plan_args'

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/NSLS-II/bluesky/pull/531/files/a82f6719047f0c3e3a3e3ecd8bc81e213a95e9c5#r73059470>, or mute the thread<https://github.com/notifications/unsubscribe-auth/ACLIrqxvi_8MPmeVg3HVp4DacdrB2O5Hks5qbmmagaJpZM4JZ8Ls>.